discord-readme.md

```diff
@@ -59,7 +59,7 @@ COMMUNITY DISCUSSION:
 
 -> #organisation: our community's GitHub org
 
--> #good-first-issues: get started with open source and get your :green_square: - opportunities are provided for new contributors - discussions not to happen here as this is just for people to share links, please discuss in #first-timers
+-> #good-first-issues: get started with open source and get your green squares opportunities are provided for new contributors - discussions not to happen here as this is just for people to share links, please discuss in #first-timers

```

Comment:
Is this change really needed?

Reply:
maybe not, I was trying to do some update on the readme according to what Eddie wrote on discord. Maybe I didn't get what it was all about.

Reply:
I would leave it with the emoji maybe adding an explanation like :green_square:  (green square) opertunity.

Reply:
I agree with @Vyvy-vi ! @schmelto That is also not necessary but if more descriptive is need then adding it is a good option

Reply:
I can't remember what I wrote on Discord 🤦‍♂️ . I think the combination of the icon and the text is more descriptive as suggested by the community 👍

Reply:
you're right.